File: app/src/main/java/com/aslansari/hypocoin/viewmodel/account/AccountUIModel.java
package com.aslansari.hypocoin.viewmodel.account;

import com.aslansari.hypocoin.repository.model.Account;

public class AccountUIModel {

    public boolean isLoading;
    public boolean isLoggedIn;
    public Account account;

    public static AccountUIModel idle() {
        // TODO create appropriate constructors
        AccountUIModel model = new AccountUIModel();
        model.account = null;
        model.isLoading = false;
        model.isLoggedIn = false;
        return model;
    }


}

File: app/src/main/java/com/aslansari/hypocoin/viewmodel/account/AccountUIModel.kt
package com.aslansari.hypocoin.viewmodel.account

import com.aslansari.hypocoin.repository.model.Account

class AccountUIModel {
    var isLoading = false
    var isLoggedIn = false
    var account: Account? = null

    companion object {
        @JvmStatic
        fun idle(): AccountUIModel {
            // TODO create appropriate constructors
            val model = AccountUIModel()
            model.account = null
            model.isLoading = false
            model.isLoggedIn = false
            return model
        }
    }
}